jobqueue/src/main/java/com/path/android/jobqueue/Job.java

```diff
@@ -76,6 +78,7 @@ public final long getDelayInMs() {
     private void writeObject(ObjectOutputStream oos) throws IOException {
         oos.writeBoolean(requiresNetwork);
         oos.writeObject(groupId);
+        oos.writeObject(singleId);
```

Comment:
normally we should add this to the end of the list. On the other hand, 2.0 is backward incompatible so does not matter, it will clear the database.